<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileManagerBarClass</class>
 <widget class="QWidget" name="FileManagerBarClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>838</width>
    <height>519</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>519</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:#FFFFFF;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="23,0,470,0">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="ftpServerLabel_">
        <property name="styleSheet">
         <string notr="true">font:13px SIMHEI;</string>
        </property>
        <property name="text">
         <string>Ftp Server ：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ftpServerLineEdit_">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>10</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="ltp::base::PushButton" name="downLoadButton_">
        <property name="text">
         <string>下载</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ltp::base::PushButton" name="cdToParentButton_">
        <property name="text">
         <string>上级目录</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ltp::base::PushButton" name="connectButton_">
        <property name="text">
         <string>连接</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_3">
     <property name="styleSheet">
      <string notr="true">border-top: 1px solid #E6E6E6;</string>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="ltp::base::FileTableWidget" name="tableWidget_">
     <column>
      <property name="text">
       <string>名称</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>大小</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>类型</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>权限</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>更新日期</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="styleSheet">
      <string notr="true">border-top: 1px solid #E6E6E6;</string>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ltp::base::FileTableWidget</class>
   <extends>QTableWidget</extends>
   <header>base/filetablewidget.h</header>
  </customwidget>
  <customwidget>
   <class>ltp::base::PushButton</class>
   <extends>QPushButton</extends>
   <header>base/pushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
